Tests/Legacy/Shopware/Connect/Subscribers/LifecycleTestLegacy.php

```diff
@@ -12,10 +12,10 @@
 use ShopwarePlugins\Connect\Components\ConnectExport;
 use ShopwarePlugins\Connect\Components\Validator\ProductAttributesValidator\ProductsAttributesValidator;
 use Tests\ShopwarePlugins\Connect\ConnectTestHelper;
-use ShopwarePlugins\Connect\Subscribers\Lifecycle;
 use ShopwarePlugins\Connect\Components\ErrorHandler;
 
-class LifecycleTest extends ConnectTestHelper
+/** Please don't rename it because this test will fail if the classname is LifecycleTest */
+class LifecycleTestLegacy extends ConnectTestHelper
```

Comment:
Then the test is not executed at all. I tried to put var_dump('something');exit; in testUpdatePrices and it doesn't work if you run ./psh.phar local:unit command